### ISSUE TITLE
Pretrained Inception V4 to Keras Application Folder 

### ISSUE THREAD

**Issue by gowthamkpr** [keras-team/keras#17343]
Repilca of this [issue](https://github.com/tensorflow/tensorflow/issues/33749) created by @amangalampalli

Would it be possible to add Inception V4 to the Keras Applications folder?

System/User Specifics: Tensorflow 2.0 with tf.keras module

Due to recent discoveries made in neural networks and also according to many benchmarking standards, Inception V4 is one of the best networks available with very high accuracies in both Top 5 and Top 1.

Inception V4 should be included because it has been consistently proven to be one of the most accurate networks for Image Recognition. It will benefit many users who want a network with very high accuracy and also have the resources to run it.

**Comment by innat** [comment]
@divyashreepathihalli 
This ticket should be moved to keras-cv.
